style/servo/url.rs

```diff
@@ -55,10 +68,10 @@ impl CssUrl {
     pub fn parse_from_string(url: String, context: &ParserContext, _: CorsMode) -> Self {
         let serialization = Arc::new(url);
         let resolved = context.url_data.0.join(&serialization).ok().map(Arc::new);
-        CssUrl {
-            original: Some(serialization),
-            resolved: resolved,
-        }
+        CssUrl(Arc::new(CssUrlData {
+             original: Some(serialization),
+             resolved: resolved 
+            }))
```

Comment:
Don't mess indentation, and don't replace the comma with a trailing space.

Reply:
Made the fix. Can you share what formatter should I be using in my dev environment? As of now, I am using default rust-analyzer.

Reply:
Use `rustfmt style/servo/url.rs`